maxim/models/test_run.py

```
@@ -64,6 +76,16 @@ class YieldedOutputCost:
     output_cost: float
     total_cost: float
 
+    def __init__(
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)